### PR TITLE
add cloudcore watch pv and pvc

### LINKED ISSUE #1899: delete pvc and pv that edgecore sqlite cannot delete information
**What happened**:
edgecore  sqlite3 can not clear pv and pvc of deletion information beacaus  cloudcore do not send deletion message  for  edgehub 
**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:
ZIOKVDBBA3%W@B7UP4YF6ZH

**Environment**:
- KubeEdge version(e.g. `cloudcore/edgecore --version`):  
edgecore1.3.0
**CloudSide Environment**:
- Hardware configuration (e.g. `lscpu`):
- OS (e.g. `cat /etc/os-release`):
- Kernel (e.g. `uname -a`):
- Go version (e.g. `go version`): 
- Others:

**EdgeSide Environment**:
- edgecore version (e.g. `edgecore --version`):
- Hardware configuration (e.g. `lscpu`):
- OS (e.g. `cat /etc/os-release`):
- Kernel (e.g. `uname -a`):
- Go version (e.g. `go version`): 
- Others:

### PR DESCRIPTION
**Pull request by guojianyu**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:

/kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind test
> /kind failing-test
> /kind feature

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #1899

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: cloud/pkg/edgecontroller/controller/downstream.go [hunk 1]
```diff
@@ -438,6 +442,119 @@ func (dc *DownstreamController) syncEndpoints() {
 		}
 	}
 }
+func (dc *DownstreamController) syncPersistentvolumes() {
+	var operation string
+	for {
+		select {
+		case <-beehiveContext.Done():
+			klog.Warning("Stop edgecontroller downstream syncEndpoints loop")
```
Comment (daixiang0) [inline cloud/pkg/edgecontroller/controller/downstream.go:450]: Could turn down it log level?

File: cloud/pkg/edgecontroller/controller/downstream.go [hunk 2]
```diff
@@ -438,6 +442,119 @@ func (dc *DownstreamController) syncEndpoints() {
...
+			return
+		case e := <-dc.persistentvolumsManager.Events():
+			pv, ok := e.Object.(*v1.PersistentVolume)
+			if !ok {
+				klog.Warningf("Object type: %T unsupported", pv)
```
Comment (daixiang0) [inline cloud/pkg/edgecontroller/controller/downstream.go:455]: Could turn down it log level?

File: cloud/pkg/edgecontroller/controller/downstream.go [hunk 3]
```diff
@@ -438,6 +442,119 @@ func (dc *DownstreamController) syncEndpoints() {
...
+				continue
+			}
+
+			ok = true
```
Comment (daixiang0) [inline cloud/pkg/edgecontroller/controller/downstream.go:459]: why need this?

File: cloud/pkg/edgecontroller/controller/downstream.go [hunk 4]
```diff
@@ -438,6 +442,119 @@ func (dc *DownstreamController) syncEndpoints() {
...
+			klog.Warning("Stop edgecontroller downstream syncPersistentvolumclaims loop")
+			return
+		case e := <-dc.persistentvolumsManager.Events():
+			pvc, ok := e.Object.(*v1.PersistentVolumeClaim)
+			if !ok {
+				klog.Warningf("Object type: %T unsupported", pvc)
+				continue
+			}
+
+			ok = true
```
Comment (daixiang0) [inline cloud/pkg/edgecontroller/controller/downstream.go:516]: why need it?

File: cloud/pkg/edgecontroller/manager/location.go [hunk 5]
```diff
@@ -213,6 +217,46 @@ func (lc *LocationCache) DeleteEndpoints(endpoints v1.Endpoints) {
 	lc.endpoints.Delete(fmt.Sprintf("%s/%s", endpoints.Namespace, endpoints.Name))
 }
 
+// GetPersistentVolume from cache
+func (lc *LocationCache) GetPersistentVolume(name string) (v1.PersistentVolume, bool) {
```
Comment (daixiang0) [inline cloud/pkg/edgecontroller/manager/location.go:221]: Which function call it?

File: cloud/pkg/edgecontroller/controller/downstream.go (hunk 4 above)
Comment (guojianyu) [reply cloud/pkg/edgecontroller/controller/downstream.go:516]: i am sorry,i overlook.i will delete it .

File: cloud/pkg/edgecontroller/manager/location.go (hunk 5 above)
Comment (guojianyu) [reply cloud/pkg/edgecontroller/manager/location.go:221]: i will delete it .

File: cloud/pkg/edgecontroller/controller/downstream.go [hunk 6]
```diff
@@ -358,7 +362,6 @@ func (dc *DownstreamController) syncEndpoints() {
 				continue
 			}
 
-			ok = true
```
Comment (daixiang0) [inline cloud/pkg/edgecontroller/controller/downstream.go:361]: ??

Comment (guojianyu) [reply cloud/pkg/edgecontroller/controller/downstream.go:361]: i deleted

**Comment by daixiang0** [comment]
Please add some decription.

**Comment by guojianyu** [comment]
> Please add some decription.
 issue(#1899) have the description of the fix
do i need to connect this issue?
but i do not test.

**Comment by guojianyu** [comment]
this fix support provisioner local pv

**Pull request closed without merging**